netcore/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This directory does not need to carry a separate license file. The root of the repository has the appropriate license.

Reply:
removed.